autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/ExpandedCourseCard.css

```
@@ -30,6 +30,14 @@
     cursor: pointer;
 }
 
+.right-header-group {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    overflow: auto;
+    width: 100%;    
```

```suggestion
    width: 100%;
```
Why do I even have this stupid whitespace plugin all it does is trigger me whenever I see other people's code with trailing whitespace aaaaaaaaaaaaaaaaaaaaaaaa